go.mod

```diff
@@ -1,18 +1,20 @@
 module github.com/konveyor/openshift-velero-plugin
 
-go 1.19
+go 1.21
+
+toolchain go1.22.4
```

Comment:
I would match this to openshift/velero

Reply:
https://github.com/openshift/velero/blob/konveyor-dev/go.mod#L3

Reply:
Thanks will fix but in a new PR. This shouldn't be master on my remote :/